pkg/build/api/validation/validation.go

```diff
@@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
 	return allErrs
 }
 
-func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
+func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
```

Comment:
might make more sense to call the allowEnv argument "isGeneric" instead, in case the two webhooks evolve further away from each other in the future.